web/ui/react-app/src/App.tsx

```diff
@@ -7,9 +7,19 @@ import {
   Row,
 } from 'reactstrap';
 
+import './App.css';
 import PanelList from './PanelList';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { faSearch, faSpinner, faCheck } from '@fortawesome/free-solid-svg-icons';
+import { faSquare, faCheckSquare } from '@fortawesome/free-regular-svg-icons';
```

Comment:
Hmm, while I would like to just keep using the solid icons like in the rest of the app, the solid square looks a bit stupid. So I guess this is fine until we find something better :)